test/e2e/dynamic_provisioning_test.go

```diff
@@ -841,4 +841,132 @@ var _ = ginkgo.Describe("[blob-csi-e2e] Dynamic Provisioning", func() {
 		}
 		test.Run(ctx, cs, ns)
 	})
+
+	ginkgo.It("should clone a volume from an existing NFSv3 volume [nfs]", func(ctx ginkgo.SpecContext) {
```

Comment:
does nfs volume also support volume cloning?

Reply:
yes, nfs is also supported